onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2260,19 +2260,107 @@ def aten_embedding_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::embedding_bag", trace_only=True)
```

Comment:
```suggestion
@torch_op(("aten::embedding_bag", "aten::_embedding_bag"), trace_only=True)
```

We can register the aten-private version as well because we happen to see them in actual models. It is essentially the same logic with contiguous input assumptions: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml#L2219

Reply:
I wonder if we can skip trae_only and let dispatcher do the work.

Reply:
And `aten::_embedding_bag_forward_only` as well.

Reply:
Both aten private versions have `padding_idx=-1`. 

`aten::embedding_bag.padding_idx` also has this argument, but the default might be slightly different `int? padding_idx`.

https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml#L2247

Reply:
Thanks for the info! I’m inclined to suggest we just use -1 and ignore the None case for ease of implementation